content/api/sending-domains.apib

```diff
@@ -6,278 +6,313 @@ description: Manage sending domains, which are used to indicate who an email is
-|verification_mailbox_verify | boolean | Request an email with a verification link to be sent to a nominated mailbox on the sending domain. | no | The nominated mailbox is specified in the verification_mailbox field.  The mailbox can be any valid mailbox for the domain other than "postmaster" or "abuse".  Not available in <span class="label label-warning"><strong>Enterprise</strong></span> |
-|verification_mailbox | string | The nominated mailbox email address local part to be used when requesting email with a verification link be sent. | no | Required if "verification_mailbox_verify" = true. Not available in <span class="label label-warning"><strong>Enterprise</strong></span> |
-|postmaster_at_verify | boolean | Request an email with a verification link to be sent to the sending domain's postmaster@ mailbox. | no | |
-|abuse_at_verify | boolean | Request an email with a verification link to be sent to the sending domain's abuse@ mailbox. | no | |
-|verification_mailbox_token | string | A token retrieved from the verification link contained in the verification email. | no | <br><br>Not available in <span class="label label-warning"><strong>Enterprise</strong></span>|
-|postmaster_at_token | string | A token retrieved from the verification link contained in the postmaster@ verification email. | no | |
-|abuse_at_token | string | A token retrieved from the verification link contained in the abuse@ verification email. | no | |
+#### Verify via DNS
 
-### DNS Attributes
+To verify your DKIM or CNAME, include `dkim_verify` or `cname_verify` in the request. SparkPost will check against and verify the associated DNS record type for the specified sending domain.
 
-| Field         | Type     | Description                           |
-|------------------------|:-:       |---------------------------------------|
-|dkim_record | string | DNS DKIM record for the registered Sending Domain |
-|cname_record | string | DNS CNAME record for the registered Sending Domain |
-|dkim_error | string | Error message describing reason for DKIM verification failure |
-|cname_error | string | Error message describing reason for CNAME verification failure |
+The request will return an extra attribute, `dns`, which contains the DNS values or error if there was a problem validating the either the DKIM or CNAME.
 
-## Create [/sending-domains]
+##### DNS Object
 
-### Create a Sending Domain [POST]
++ Data Structure: Attributes
+    + dkim_record (string) - DNS DKIM record for the registered sending domain
+    + cname_record (string) - DNS CNAME record for the registered sending domain
+    + dkim_error (string) - Error message describing reason for DKIM verification failure
+    + cname_error (string) - Error message describing reason for CNAME verification failure
 
-Create a sending domain by providing a **sending domain object** as the POST request body.
+#### Verify via mailbox
```

Comment:
I think this might deserve splitting the requests into two (similar to templates):
- Verify via DNS
- Verify via Mailbox 

with the relevant request body attributes in each one